components/kyma-environment-broker/internal/process/new_provisioning/staged_manager.go

```diff
@@ -0,0 +1,241 @@
+package new_provisioning
```

Comment:
I'd change the name of the packag. it is not about provisioning, maybe `process`? or maybe you have a better idea